daemon/api.go

```diff
@@ -845,7 +845,7 @@ func snapUpdateMany(inst *snapInstruction, st *state.State) (msg string, updated
 	case 0:
 		// not really needed but be paranoid
```

Comment:
Is that message right?  It inferred that it would never happen, but it does happen. Is something broken that we should worry about, or is this a normal situation?  The message should go if the latter.